### PR TITLE
docs: update issue and pr templates

### LINKED ISSUE #6583: docs: Make issue forms consistent
### Improved or typo fixed.

The current issue forms are inconsistent with each different scenarios.

### Solution

Make the issue form consistent across all the different issue types and scenarios.

### PR DESCRIPTION
**Pull request by navendu-pottekkat**
Signed-off-by: Navendu Pottekkat <navendupottekkat@gmail.com>

### What this PR does / why we need it:

This PR updates the issue and PR templates and makes it consistent.

Fixes #6583

### CODE REVIEW

**Comment by navendu-pottekkat** [comment]
Hmm, seems like there are a lot of CI checks being run which are unnecessary. We may need to blacklist changes to this path so that it doesn't trigger these workflows.

File: .github/ISSUE_TEMPLATE/request_help.yml [hunk 1]
```diff
@@ -1,35 +1,36 @@
...
     id: environment
     attributes:
       label: Environment
-      description: Request help without environment information will be ignored or closed.
+      description: Share your environment details. Reports without proper environment details will likely be closed.
       value: |
-        - apisix version (cmd: `apisix version`):
-        - OS (cmd: `uname -a`):
-        - OpenResty / Nginx version (cmd: `nginx -V` or `openresty -V`):
-        - etcd version, if have (cmd: run `curl http://127.0.0.1:9090/v1/server_info` to get the info from server-info API):
```
Comment (spacewander) [inline .github/ISSUE_TEMPLATE/request_help.yml:30]: Why remove the `get the info from server-info API`?

Comment (navendu-pottekkat) [reply .github/ISSUE_TEMPLATE/request_help.yml:30]: `etcd version` implies that you have to get it. And the API is self documented in my opinion and it is obvious what it does. So, an additional sentence might be redundant? What do you think? Should I add back that sentence?

Comment (spacewander) [reply .github/ISSUE_TEMPLATE/request_help.yml:30]: I see.

**Review by spacewander — APPROVED** [review]

**Comment by navendu-pottekkat** [comment]
> Hmm, seems like there are a lot of CI checks being run which are unnecessary. We may need to blacklist changes to this path so that it doesn't trigger these workflows.

@spacewander What do you think about this?

**Comment by spacewander** [comment]
> > Hmm, seems like there are a lot of CI checks being run which are unnecessary. We may need to blacklist changes to this path so that it doesn't trigger these workflows.
> 
> @spacewander What do you think about this?

There are ignores:
https://github.com/apache/apisix/blob/a490e0a2570d916921b4a4e50f3c308291ba7dd9/.github/workflows/build.yml#L7
but not in a fine grain size

**Review by tokers — APPROVED** [review]

**Pull request merged**